src/tree/box.js

```diff
@@ -97,7 +104,24 @@ export default function box(
         selections.select(node, sel, selectionState);
       }
     })
-    .html(d => card(d.data, cardStyling, sel, selectionState));
+    .html(d => card(d.data, cardStyling, sel, selectionState))
+    .on('mouseenter', d => {
+      if (allowInteractions) {
```

Comment:
Actually we should use `!constraints.passive` here. Tooltips should be shown play mode for snapshots. And passive is undefined in play mode, where active is true